### ISSUE TITLE
Update index?

### ISSUE THREAD

**Issue by maccman** [trivio/common_crawl_index#23]
Not sure if this is the right place to create a ticket, but I'm wondering if it's possible to update the url index to the March 2014 crawl?

**Comment by aparup** [comment]
2014 crawl index is work in progress now. You will see an update soon.

On Tue, May 13, 2014 at 4:23 PM, Alex MacCaw notifications@github.comwrote:

> Not sure if this is the right place to create a ticket, but I'm wondering
> if it's possible to update the url index to the March 2014 crawl?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/trivio/common_crawl_index/issues/23
> .

## 

Aparup

**Comment by maccman** [comment]
Awesome - thanks @aparup

**Comment by aparup** [comment]
Any update on index?

Thanks in advance
--Raviteja

**Comment by aparup** [comment]
Still work in progress. Should provide an ETA soon.

Aparup

On Thu, Jun 5, 2014 at 12:48 AM, Raviteja reddy <rtr.ravitejareddy@gmail.com

> wrote:
> 
> Any update on index?
> 
> Thanks in advance
> --Raviteja
> 
> On Wednesday, 14 May 2014 04:55:28 UTC+5:30, Aparup Banerjee wrote:
> 
> > 2014 crawl index is work in progress now. You will see an update soon.
> > 
> > On Tue, May 13, 2014 at 4:23 PM, Alex MacCaw notifi...@github.com
> > wrote:
> > 
> > > Not sure if this is the right place to create a ticket, but I'm
> > > wondering if it's possible to update the url index to the March 2014 crawl?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Ftrivio%2Fcommon_crawl_index%2Fissues%2F23&sa=D&sntz=1&usg=AFQjCNFmoKeIAzKxHZvI8JGbLKae50GCSQ
> > > .
> > 
> > ## 
> > 
> > Aparup
> > 
> >  --
> > You received this message because you are subscribed to the Google Groups
> > "Common Crawl" group.
> > To unsubscribe from this group and stop receiving emails from it, send an
> > email to common-crawl+unsubscribe@googlegroups.com.
> > To post to this group, send email to common-crawl@googlegroups.com.
> > Visit this group at http://groups.google.com/group/common-crawl.
> > For more options, visit https://groups.google.com/d/optout.

## 

Aparup

**Comment by aparup** [comment]
Any update on the index?

**Comment by maccman** [comment]
Just going to re-open this issue until an update is posted.

**Comment by maccman** [comment]
If there's anything I can do to help, let me know! More than happy to contribute.

**Comment by maccman** [comment]
New CommonCrawl data out - would be great to update the url index.

http://commoncrawl.org/august-2014-crawl-data-available/

**Comment by mattpowell** [comment]
Ditto wrt to helping/contributing! :)

**Comment by aparup** [comment]
I just cloned the repo and it appears it is still pointing at the 2012
corpus (
https://aws-publicdatasets.s3.amazonaws.com/common-crawl/projects/url-index/url-index.1356128792).
Is there a newer index available? If so, where can I find it? If not, is
anyone working on generating one?

Thanks,

Kevin

**Comment by aparup** [comment]
That's true. New Index is still WIP, should be out soon. Also the index of
2012 which is generated is half (roughly) of the total corpus.

Aparup

On Sun, Oct 12, 2014 at 4:02 PM, Kevin Fink kevin@fink.com wrote:

> I just cloned the repo and it appears it is still pointing at the 2012
> corpus (
> https://aws-publicdatasets.s3.amazonaws.com/common-crawl/projects/url-index/url-index.1356128792).
> Is there a newer index available? If so, where can I find it? If not, is
> anyone working on generating one?
> 
> Thanks,
> 
> Kevin
> 
> On Wed, Jun 25, 2014 at 4:15 AM, Denny Britz dennybritz@gmail.com wrote:
> 
> > Any update on the index?
> > 
> > On Wednesday, May 14, 2014 6:25:28 AM UTC+7, Aparup Banerjee wrote:
> > 
> > > 2014 crawl index is work in progress now. You will see an update soon.
> > > 
> > > On Tue, May 13, 2014 at 4:23 PM, Alex MacCaw notifi...@github.com
> > > wrote:
> > > 
> > > > Not sure if this is the right place to create a ticket, but I'm
> > > > wondering if it's possible to update the url index to the March 2014 crawl?
> > > > 
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > https://github.com/trivio/common_crawl_index/issues/23.
> > > 
> > > ## 
> > > 
> > > Aparup
> > > 
> > >  --
> > > You received this message because you are subscribed to the Google Groups
> > > "Common Crawl" group.
> > > To unsubscribe from this group and stop receiving emails from it, send an
> > > email to common-crawl+unsubscribe@googlegroups.com.
> > > To post to this group, send email to common-crawl@googlegroups.com.
> > > Visit this group at http://groups.google.com/group/common-crawl.
> > > For more options, visit https://groups.google.com/d/optout.
> 
>  --
> You received this message because you are subscribed to the Google Groups
> "Common Crawl" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to common-crawl+unsubscribe@googlegroups.com.
> To post to this group, send email to common-crawl@googlegroups.com.
> Visit this group at http://groups.google.com/group/common-crawl.
> For more options, visit https://groups.google.com/d/optout.

## 

Aparup

**Comment by aparup** [comment]
Is there something more definite than "soon"? The timing will determine
whether I use Common Crawl for a current project or go a different
direction.

Which crawl will it cover? The latest (Aug 2014)? Also, I'm assuming, but
want to verify, that this index will include the entire crawl.

Thanks!

Kevin

**Comment by aparup** [comment]
+1

**Comment by srobertson** [comment]
I'm confused whose posting what on this issue. Maybe this is linked to the google forum?

Last update I heard is we found a new volunteer to generate the index and hopefully engineer so that it's generated on an ongoing basis.

Sorry to be dark had a new addition to the family and an increased load in my professional life. Not sure if when I have time to personally oversee this. 

What I can do if anyone is interested is organize a hack night where we go over the code used to generate the index if there's interest.

**Comment by aparup** [comment]
Last update I heard is we found a new volunteer to generate the index and 
hopefully engineer so that it's generated on an ongoing basis.

Sorry to have gone dark here for so long.  Had a new addition to the family 
and an increased load in my professional life. Not sure if when I have time 
to personally oversee the index project.

What I can do, if there's enough interest, is organize a meetup  (i'll 
provide the beer,pizza and location) where we go over the code used to 
generate the index, discuss how to improve it and maybe hack a little bit. 
At the very least we can spread the knowledge around.

Any interest?

**Comment by aparup** [comment]
I'm interested, but not knowing where everyone is physically, it might have
to be a virtual meetup. I'm usually physically in Kirkland, WA (although
I'm in Denver, CO at the moment, and sometimes get to Austin, TX).

Kevin

**Comment by aparup** [comment]
I'd be interested in that meetup (I'm in Los Angeles).

**Comment by aparup** [comment]
On Thu, Oct 16, 2014 at 11:47:02AM -0700, Scott Robertson wrote:

> What I can do, if there's enough interest, is organize a meetup  (i'll 
> provide the beer,pizza and location) where we go over the code used to 
> generate the index, discuss how to improve it and maybe hack a little bit. 
> At the very least we can spread the knowledge around.

It's great to see all of this interest in the urlindex! Given the
geographic distribution of the interested folks, I think we're going
to have to do a Google Hangout or equivalent -- something where we can
share a screen of source code and talk.

-- greg

**Comment by aparup** [comment]
Please count me in for a hangout too and keep us posted!

**Comment by oskarpearson** [comment]
Hi. I was wondering what the cost is to generating the index on AWS. ie - is it going to take dozens of machines multiple days? I'm thinking of trying to look into this, but I'm not sure if it's a $100s or $1000s project.

Thanks!

**Comment by aparup** [comment]
Is there any update on 2014 crawl index ? And how to generate and save the 
index files ? Can I do something even I'm far away?

--Hector

在 2014年5月14日星期三UTC+8上午7时25分28秒，Aparup Banerjee写道：

> 2014 crawl index is work in progress now. You will see an update soon.
> 
> On Tue, May 13, 2014 at 4:23 PM, Alex MacCaw <notifi...@github.com

**Comment by aparup** [comment]
Hi Hector

There was a group call about the URL Index on the 19th of November and a
status update was given then.  You can see a summary of the call on this
thread https://groups.google.com/forum/#!topic/common-crawl/jN8OwUeztE8.

Thanks for offering to help! We are planning another group call about the
URL Index very soon.  Would you like to join the next call to talk about
how you might contribute?

Lisa

Lisa Green
Director, The Common Crawl Foundation
www.commoncrawl.org

**Comment by aparup** [comment]
Hi Lisa,

Just joined the list. Please keep us posted on the next call, I would like 
to see if I can contribute to the code.

Kurman

**Comment by popol1991** [comment]
Two years later and this issue is still open... Is there any news for the index?

**Comment by wumpus** [comment]
This project is not moving forward; you can find an index made in the traditional CDX way at 

http://index.commoncrawl.org/

This other index is up-to-date.

**Comment by dportabella** [comment]
any update to this project to use the latest dataset, and use the WARC, WAT and/on WET files?

I see this other project: http://index.commoncrawl.org/
it allows to query the segments for a given url prefix, but there is not a solution to copy the files as in with `bin/remote_copy`.

**Comment by wumpus** [comment]
You can write a replacement for bin/remote_copy and the other index a lot faster than reviving this one. I should probably put in a pull request to update the docs for this project to explain these issues.

**Comment by dportabella** [comment]
I didn't understand what you said. you mean that it's easier to write a `remote_copy` script for the index.commoncrawl.org project, rather than updating this common_crawl_index project?

I am trying to do so, but I get a problem here: https://github.com/ikreymer/cdx-index-client/issues/3

**Comment by wumpus** [comment]
Yes, that's what I mean. See my comment about your other bug on the other bug.
